### PR TITLE
Vy/schemas/file

### PR DESCRIPTION
**Pull request by praktikon**

### CODE REVIEW

File: backend/models/Mongoose Schema - event.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "title": {
+        "type": "String"
+    },
+    "project": {
+        "type": "Schema.Types.ObjectId"
```
Comment (DmitrijDN) [inline backend/models/Mongoose Schema - event.js]: ref is absent

File: backend/models/Mongoose Schema - event.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "description": {
+        "type": "String"
+    },
+    "startDate": {
+        "type": "Date"
+    },
+    "endDate": {
+        "type": "Date"
+    },
+    "participants": [],
```
Comment (DmitrijDN) [inline backend/models/Mongoose Schema - event.js]: array of what

File: backend/models/Mongoose Schema - file.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "url": {
+        "type": "String"
+    },
+    "owner": {
+        "type": "Schema.Types.ObjectId"
```
Comment (DmitrijDN) [inline backend/models/Mongoose Schema - file.js]: ref

File: backend/models/Mongoose Schema - file.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    },
+    "creationDate": {
+        "type": "Date"
+    },
+    "allowedTo": []
```
Comment (DmitrijDN) [inline backend/models/Mongoose Schema - file.js]: array of what

**Comment by ihorhabshiy** [comment]
ПРи робимо в девелоп.

**Pull request closed without merging**